### DISCUSSION TITLE
different base unit

### DISCUSSION THREAD

**Discussion by hofk** [boytchev/mannequin.js#27] [General]
I just recently became aware of mannequin.js and am testing to integrate it into my project. 

All tests worked great.   
My project uses a different base unit than mannequin.js Therefore I have to multiply by a factor of 0.1. 

This is not really a problem, because you can use for example `man = new Male( 0.1 );`

This works for all parts of the body except the head. That is too big. For testing I tried `man = new Male( 3 ); ` the head is considerably too small, only as thick as the neck.

I saw in Refactored mannequin.js that recent changes were made to the HeadShape in lines 311 to 314. Is that where the error may have crept in?

![2023-02-17 20 55 12](https://user-images.githubusercontent.com/27998882/219778583-780b4e06-bbf0-4e6e-80aa-c814ac408fc5.png)

![2023-02-17 20 53 27](https://user-images.githubusercontent.com/27998882/219778156-9b3d4226-72d9-491c-b433-af6fa1fa6ef7.png)

**Comment by boytchev** [comment]
The height parameter in the constructor is the age-related height. The head is scaled unproportionally to the body. This is used to make figures of children, where the head is much larger in respect to the rest of the body.

If you want to scale a figure proportionally, for example 10 times smaller, use:
```javascript
man = new Male( );
man.scale.set( 0.1, 0.1, 0.1 );
```

**Reply by hofk** [reply]
Thanks for pointing that out, I probably haven't read enough to figure that out. Makes sense!

Then there should be no further problems, I have already created a test example that matches my project.
